docs/recurring.md

```diff
@@ -0,0 +1,183 @@
+
+```html
+<label>
+  CVC
+  <div id="cvc"></div>
+</label>
+<br />
+<button id="submit">Submit</button>
+```
+
+We will mount the cvc elements into the empty divs in step 5 and create a handler for the submit button in step 6.
+
+### 4. Create the split cvc elements
+
+This creates the specified [Element](/docs#Element) objects. We specify the types as `cvc`.
+
+```js
+const cvc = Airwallex.createElement('cvc');
+```
+
+There are also additional options as a second parameter to the `createElement` function that can overwrite styles and other functions. [See docs](/docs#createElement) for more details.
+
+### 5. Mount the cvc elements
+
+Next, we need to mount the cvc element to the DOM.
+
+```js
+cvc.mount('cvc');
+```
+
+This function will append the cvc element to your divs with ids `cvc`, as created above in step 3. **Ensure that there are no other elements in the document with the same ids**.
+
+Each **cvc element should only be mounted once** in a single payment flow.
+
+### 6. Add a button handler to trigger the payment request and listen to the request response.
+
+This handler is called when a customer is ready to make a payment according to the details documented in the Payment Intent, thereby confirming the Payment Intent.
+
+```js
+// STEP #6a: Add a button handler
+document.getElementById('submit').addEventListener('click', () => {
+  Airwallex.confirmPaymentIntent({
+    element: cvc, // Provide the cvc element
+    id: 'replace-with-your-intent-id', // Payment Intent ID
+    client_secret: 'replace-with-your-client-secret', // Client Secret
+    payment_consent_id: 'replace-with-your-consent-id' // Payment Consent id,
+  }).then((response) => {
+    // STEP #6b: Listen to the request response
+    /* Handle response */
+    window.alert(JSON.stringify(response));
+  });
+});
+```
+
+`Airwallex.confirmPaymentIntent` will take the cvc element you mounted and confirm the payment details entered to the payment intent (provided by the `id` prop). A `client_secret` must be provided to authenticate the checkout process.
+
+More details about the `confirmPaymentIntent` function can be found [here](/docs#confirmPaymentIntent).
+
+### 7. Add an `onReady` event listener to handle events when the element is mounted
+
+```js
+window.addEventListener('onReady', (event) => {
+  /*
+    ... Handle event
+  */
+  window.alert(event.detail);
+});
+```
+
+This can be used to set a loading state as the checkout screen is being prepared.
+
+### 8. Add an `onChange` event listener to listen to the changes in each of the input fields
+
+```js
+window.addEventListener('onChange', (event) => {
+  /*
+    ... Handle event
+  */
+  window.alert(event.detail);
+});
+```
+
+`event` will return an object with the field name and whether this field is completed (valid). This can help with validating the fields before users can trigger the submit button to prevent any validation errors.
+
+### 9. Beautify and deploy!
```

Comment:
Do we need to add a full demo of a recurring flow solution?

Reply:
Added a demo of split card.